src/HaskellCI/Config/History.hs

```diff
@@ -72,6 +72,8 @@ configHistory =
     -- https://github.com/haskell-CI/haskell-ci/issues/766
     , ver 0 19 20250115 := \cfg -> cfg
         & field @"cfgCabalInstallVersion" ?~ C.mkVersion [3,12,1,0]
+    , ver 0 19 20250315 := \cfg -> cfg
+        & field @"cfgGhcupVersion" .~ C.mkVersion [0,1,40,0]
```

Comment:
will need to update the script to use aliases for prerelease and vanilla channels, those have 0.0.8 version (and path overall) hardcoded.

Reply:
```
[ Warn  ] New metadata version detected
[ ...   ]     old URI: https://raw.githubusercontent.com/haskell/ghcup-metadata/master/ghcup-prereleases-0.0.8.yaml
[ ...   ]     new URI: https://raw.githubusercontent.com/haskell/ghcup-metadata/refs/heads/master/ghcup-prereleases-0.0.9.yaml
```